packages/blocks/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 5.1.0 (Unreleased)
+
+### New feature
+
+- `isBlockContentValid` function has been added ([#10891](https://github.com/WordPress/gutenberg/pull/10891)).
```

Comment:
Nit: We have some precedent of the adjective coming first in boolean "is" functions, e.g. `isReusableBlock`, so for consistency's sake, `isValidBlockContent` might be the more consistent ordering.

Reply:
I see also `isValidElement`, let's rename. 👍